game.js

```diff
@@ -0,0 +1,38 @@
+const FrameClass = require('./frame');
+
+class Game {
+  constructor(Frame = FrameClass) {
+    this.frames = [];
+    for (let i = 0; i < 10; i++) {
+      this.frames.push(new Frame());
+    }
+  }
+
+  roll(points) {
+    for (let i = 0; i < 9; i++) {
+      if (this.frames[i].rolls.length < 2 && this.frames[i].isStrike() === false) {
+        this.frames[i].addRoll(points);
+        return;
+      }
+    }
+    this.frames[9].addRoll(points);
+  }
+
+  addBonuses() {
+    this.frames.forEach((frame, index) => {
+      if (frame.isSpare() == true) {
+        frame.bonus = this.frames[index + 1].rolls[0];
+      } else if (frame.isStrike() == true) {
+        frame.bonus = this.frames[index + 1].rolls[0] + this.frames[index + 1].rolls[1];
```

Comment:
would line 26 catch two rolls for a strike's bonus points if the following frame was also a strike? i.e. rolls[1] did not exist

(The bonus for that frame is the number of pins knocked down by the next two rolls. That would be the next frame, unless the player rolls another strike.)